OpenRA.Mods.Common/Widgets/Logic/Lobby/SpawnSelectorTooltipLogic.cs

```diff
@@ -18,13 +18,13 @@ namespace OpenRA.Mods.Common.Widgets.Logic
 	public class SpawnSelectorTooltipLogic : ChromeLogic
 	{
 		[TranslationReference]
-		static readonly string DisabledSpawn = "disabled-spawn";
+		static readonly string DisabledSpawn = "tooltip-disabled-spawn";
 
 		[TranslationReference]
-		static readonly string AvailableSpawn = "available-spawn";
+		static readonly string AvailableSpawn = "tooltip-available-spawn";
 
 		[TranslationReference("team")]
-		static readonly string TeamNumber = "team-number";
+		static readonly string TeamNumber = "label-team-name";
```

Comment:
we query for team 0, even though we don't use it, since the label is visible only when `playerTeam > 0`. It seems incorrect

Reply:
Adressed in #20269